ui/packages/tidb-dashboard-for-op/src/apps/Metrics/context.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  IMetricsDataSource,
+  IMetricsContext,
+  ReqConfig
+} from '@pingcap/tidb-dashboard-lib'
+
+import client from '~/client'
+
+class DataSource implements IMetricsDataSource {
+  getTiDBTopology(options?: ReqConfig) {
+    return client.getInstance().getTiDBTopology(options)
+  }
+
+  getStoreTopology(options?: ReqConfig) {
+    return client.getInstance().getStoreTopology(options)
+  }
+
+  getPDTopology(options?: ReqConfig) {
+    return client.getInstance().getPDTopology(options)
+  }
```

Comment:
For `Metrics` App, we don't need these APIs. I think we just need `metricsQueryGet`.